docs/ADR-72-ab-testing.md

```diff
@@ -0,0 +1,139 @@
+# A/B testing
+
+## Context and Problem Statement
+
+Given the amount of new features we need a way to measure the impact that each one has on the use of our products and A/B testing is the way we use to decide between two or more versions of a new feature and/or compared with the current one. This ADR describe how we measure A/B testings in our products.
+
+## Main objectives
+
+- sanitize metrics
+- allow data team to develop, monitor and improve dashboards and tools decoupled from the implementation on each app
+- allow any team to implement a/b testings on their environment without dependencies with other teams
+- leverage over already implemented and tested tools
+
+## Current implementation
+
+To setup an A/B testing we use Feature Flags as assignation tool and Segment as measurement tool.
+
+### Feature Flags: Setting up a new A/B testing
+
+On this standard an A/B is defined as a feature flag that ends with `_variant` with each of the possible variants defined as variant and the activation defined as your application required.
+
+> Please note that if you need to test a new variant against the current one it is preferable to use a variant with two options (`enabled` and `disabled`) instead of the activation because otherwise you won't be able to differentiate (and measure) when your a/b is active or just should show the current version.
```

Comment:
```suggestion
> Please note that if you need to test a new variant against the current one it is preferable to use a variant with two options (`enabled` and `disabled`) instead of the activation because otherwise, you won't be able to differentiate (and measure) when your a/b is active or just should show the current version.
```